api/src/main/java/jakarta/persistence/PersistenceConfiguration.java

```diff
@@ -183,6 +321,7 @@ public PersistenceConfiguration mappingFile(String name) {
     }
 
     /**
+     * The configured XML mapping files.
      * @return all configured mapping file names
```

Comment:
dtto `s/file/resource` for consistency with the spec/schema to avoid confusion about how the _String_ is to be treated